src/components/Testimonials/Testimonials.js

```diff
@@ -62,3 +70,25 @@ Testimonials.defaultProps = {
 }
 
 export default withStyles(s)(Testimonials)
+
+function CustomNextArrow(props) {
+  const { className, style, onClick } = props
+  return <div className={className} style={{ ...style, display: 'block' }} onClick={onClick} />
+}
+
+CustomNextArrow.propTypes = {
+  className: PropTypes.string.isRequired,
+  style: PropTypes.string.isRequired,
+  onClick: PropTypes.func.isRequired,
+}
+
+function CustomPrevArrow(props) {
+  const { className, style, onClick } = props
+  return <div className={className} style={{ ...style, display: 'block' }} onClick={onClick} />
+}
+
+CustomPrevArrow.propTypes = {
```

Comment:
This is a stateless component. use the same format as `Aavatar.js`, look how I'm declaring the component there, and how I use `Props`